dotcom-rendering/index.d.ts

```diff
@@ -620,21 +635,13 @@ type FEContainerType =
 	| 'news/most-popular';
 
 type FEContainerPalette =
-	| `Branded`
```

Comment:
I think we definitely need to keep `Branded` for the Labs container!

<img width="1007" alt="image" src="https://user-images.githubusercontent.com/76776/166442387-6a9ca622-350f-45a3-8c2d-83889dd38cc8.png">

Reply:
I think that there will defo be other 'non palette' tags that we will eventually bring into this pattern. Branded seems like a good example. But right now that isn't included as a, er, classic palette, it's a palette like thing, but doesn't exist in [this file](https://github.com/guardian/interactive-atom-container-colours/blob/3733462f6a475dfa49c28320041633d8b426aaaa/shared/css/_variables.scss).

I think for these types of tags we probably need to do an investigation into each one and include them as separate PRs. Those PRs may or may not end up with us adding a new 'DCR' palette type. Branding looks like it could end up there but there's a chance we learn something unexpected. Eg. That container has a lot of additional use cases that we don't support right now, so maybe we end up creating a new container component entirely? I'm advocating for this but using it as an example for how we want to dig into these other types more before committing them to a pattern

Reply:
Interesting… That’s an entirely valid reason to think about this edge case in more details. I think that having a way to style containers will be needed for at `Branded` and `Video`, unless the latter is an altogether different Container type.

In `frontend`, here’s what I found about the supported container types for the `Labs` container: https://cs.github.com/guardian/frontend/blob/09669f922da66350190fff5e44d0dda968db3a81/common/app/views/fragments/commercial/containers/paidContainer.scala.html?q=org%3Aguardian+labs-container#L30-L43

Reply:
It's interesting that the container types would be restricted here, in the rendering layer. I would say these sorts of things should be part of the Fronts config tool. Ideally, the rendering layer would have minimal business logic and just do what is asked of it 🤔